src/test/java/com/google/sps/PromotionQuizResultsTest.java

```diff
@@ -0,0 +1,76 @@
+package com.google.sps;
+
+import com.google.sps.data.ProcessPromotionQuizResults;
+import com.google.sps.data.QuestionChoice;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests that the submission of choices for the CareerQuizServlet returns recommended path that is
+ * associated to the majority of choices.
+ */
+@RunWith(JUnit4.class)
+public final class PromotionQuizResultsTest {
+
+  private static final String CAREER_1 = "Web Developer";
+  private static final String CAREER_2 = "Program Manager";
+  private static final String CAREER_3 = "Software Engineer";
+  private static final String CHOICE_1 = "choice1";
+  private static final String CHOICE_2 = "choice2";
+  private static final String CHOICE_3 = "choice3";
+  private static final String CHOICE_4 = "choice4";
+  private static final String CHOICE_5 = "choice5";
+
+  @Test
+  public void proportionAcceptableChoicesAreOverThreshold_GetsPromoted() {
+    List<QuestionChoice> userChoices =
+        Arrays.asList(
+            new QuestionChoice(CHOICE_1, CAREER_1, /* isAcceptableChoice = */ true),
```

Comment:
I little tip, if you name two static variables:
```
private static final ACCEPTABLE_CHOICE = true;
private static final NOT_ACCEPTABLE_CHOICE = false;
```

Then you can do 
```
new QuestionChoice(CHOICE_1, CAREER_1, ACCEPTABLE_CHOICE)
```

Reply:
Nifty, changed.